### DISCUSSION TITLE
Xdrip and dexcom one connection

### DISCUSSION THREAD

**Discussion by markori** [NightscoutFoundation/xDrip#2747] [Q&A]
Dear all, 
Please help us with our xdrip and dexcom one transmitter. We are unable to connect. We uploaded detail logs

Thank you
![Screenshot_20230325-210639460_1](https://user-images.githubusercontent.com/43321392/227739378-3ac9a855-f99c-44c2-8d10-4c7fd50488d3.jpg)
![Screenshot_20230325-212136069_1](https://user-images.githubusercontent.com/43321392/227739754-d09a9bd1-9858-4b71-b588-d868dc368895.jpg)

**Comment by iwascu** [comment]
Your version of XDRIP not WORKING on dexcom ONE. tray 12.01.2023 version...is working https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.01.12

**Comment by kresimirjedud** [comment]
There is defenitely an issue with newer versions and Dexcome one. I can
confirm this version is OK so try with it:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.02.08


Regards,

ned, 26. ožu 2023. 00:03 iwascu ***@***.***> je napisao:

**Reply by Navid200** [reply]
Exactly which version is the newest one that still works, March 6?
